fixtures/shehds/mini-led-spot-beam-10w-light.json

```
@@ -0,0 +1,182 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini LED Spot Beam 10W Light",
+  "shortName": "mini-led-spot",
+  "categories": ["Color Changer", "Moving Head"],
+  "meta": {
+    "authors": ["WIlliam Tessier"],
+    "createDate": "2023-03-22",
+    "lastModifyDate": "2023-03-22"
+  },
+  "comment": "Mini 10W small moving head light.",
+  "links": {
+    "manual": [
+      "https://www.shehds.com/u_file/1907/file/ea38ccee37.pdf"
+    ],
+    "productPage": [
+      "https://www.shehds.com/products/mini-led-spot-beam-10w-lighting"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=WrfsHyYv8QA"
+    ]
+  },
+  "physical": {
+    "dimensions": [127, 135, 127],
```

```suggestion
    "dimensions": [145, 120, 260],
```

The webpage and manual disagree.  This is what the manual says.  The webpage seems unreliable in other aspects (e.g., 7/15 DMX channels, when their own downloads have only 13ch modes).